chroma-manager/tests/integration/core/api_testcase_with_test_reset.py

```diff
@@ -835,9 +835,9 @@ def create_zpools(self):
                                       server0['fqdn'],
                                       'create zfs device %s' % zfs_device)
 
-                self.execute_commands(zfs_device.release_commands,
-                                      server0['fqdn'],
-                                      'export zfs device %s' % zfs_device)
+#                self.execute_commands(zfs_device.release_commands,
+#                                      server0['fqdn'],
+#                                      'export zfs device %s' % zfs_device)
```

Comment:
Still need this?

Reply:
will remove that later in the stack in this commit https://github.com/intel-hpdd/intel-manager-for-lustre/pull/467/commits/dfa08eb81c98ef162a86ea7be42b0edd429f6881